codalab/apps/web/bundles.py

```diff
@@ -202,32 +203,35 @@ def upload_bundle(self, source_file, bundle_type, worksheet_uuid):
             return new_bundle_uuid
 
         def add_worksheet_item(self, worksheet_uuid, bundle_uuid):
-            '''
+            """
             Add bundle uuid to the given worksheet.
-            '''
+            """
             self.client.add_worksheet_item(worksheet_uuid, worksheet_util.bundle_item(bundle_uuid))
 
         def parse_and_update_worksheet(self, uuid, lines):
-            '''
+            """
             Replace worksheet |uuid| with the raw contents given by |lines|.
-            '''
+            """
             worksheet_info = self.client.get_worksheet_info(uuid, True)
             new_items, commands = worksheet_util.parse_worksheet_form(lines, self.client, worksheet_info['uuid'])
             self.client.update_worksheet_items(worksheet_info, new_items)
             # Note: commands are ignored
 
         def get_bundle_contents(self, uuid):
-            '''
+            """
             If bundle is a single file, get file contents.
             Otherwise, get stdout and stderr.
             For each file, only return the first few lines.
-            '''
+            """
             def get_lines(name):
                 lines = self.head_target((uuid, name), self.HEAD_MAX_LINES)
                 if lines is not None:
                     import base64
                     lines = ''.join(map(base64.b64decode, lines))
-                return lines
+                try:
```

Comment:
Maybe we should move this to the codalab-cli repo and have it be a utility function that both the BundleCLI and this file can call.